### PR TITLE
Added icons

### PR DESCRIPTION
**Pull request by Riley142**

### CODE REVIEW

**Comment by Riley142** [comment]
Added icons in .md file - You have to npm install them first:
cd app
npm install remixicon --save

File: app/src/icons.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+To install cd app and run npm install remixicon --save
```
Comment (nmunro) [inline app/src/icons.md]: This isn't necessary, if it's installed to package-lock.json (which it should be) other developers need only run 'npm install'

Comment (Riley142) [reply app/src/icons.md]: Oh cool! Thank You :)

**Review by Riley142 — COMMENTED** [review]
remove the remixicons install

**Review by jadavey91 — APPROVED** [review]

**Review by lisarawlings — APPROVED** [review]

**Review by boothscript — APPROVED** [review]
Awesome!

**Pull request merged**